test/GroupByTest.cpp

```diff
@@ -642,29 +759,41 @@ TEST_F(GroupBySpecialCount, computeGroupByForFullIndexScan) {
     Id idOfX;
     Id idOfY;
     Id idOfZ;
+    Id idOfA;
+    Id idOfB;
+    Id idOfC;
     qec->getIndex().getId("<x>", &idOfX);
     qec->getIndex().getId("<y>", &idOfY);
     qec->getIndex().getId("<z>", &idOfZ);
+    qec->getIndex().getId("<a>", &idOfA);
+    qec->getIndex().getId("<b>", &idOfB);
+    qec->getIndex().getId("<c>", &idOfC);
 
-    // Three distinct subjects.
-    ASSERT_EQ(result.size(), 3);
+    // Six distinct subjects.
+    ASSERT_EQ(result.size(), 6);
     if (includeCount) {
       ASSERT_EQ(result.numColumns(), 2);
     } else {
       ASSERT_EQ(result.numColumns(), 1);
     }
     // The test index currently consists of 6 triples.
-    EXPECT_EQ(result(0, 0), idOfX);
-    EXPECT_EQ(result(1, 0), idOfY);
-    EXPECT_EQ(result(2, 0), idOfZ);
+    EXPECT_EQ(result(0, 0), idOfA);
+    EXPECT_EQ(result(1, 0), idOfB);
+    EXPECT_EQ(result(2, 0), idOfC);
+    EXPECT_EQ(result(3, 0), idOfX);
+    EXPECT_EQ(result(4, 0), idOfY);
+    EXPECT_EQ(result(5, 0), idOfZ);
```

Comment:
You can use this update to refactor these tests using test helpers that were introduced since this was last touched:

we have in the `IndexTestHelpers` a `makeGetId()` functor, so you can (syntax details have to be looked up)
do something like: 

```
#include <gmock/gmock.h> // at top of file
...
auto getId = makeGetId(theIndex);
EXPECT_THAT(result.getColumn(0), ::testing::ElementsAre(getId("<x>", getId("<y>), ...));
```